tasks/build-tachometer-comment.js

```diff
@@ -187,4 +186,15 @@ ${body[1]}
     return comment;
 };
 
-console.log(buildTachometerComment());
+export default function main() {
```

Comment:
I wrapped the output in this function to make it easier to call this script from the command line.